docusaurus/docs/cloud/getting-started/usage-billing.md

```diff
@@ -107,11 +101,11 @@ While the project is suspended:
 
 To reactivate the project immediately, users can upgrade to a paid plan.
 
-#### Project suspension after subscription cancellation
+#### Project suspension due to billing issues
```

Comment:
Note to self (nothing to do on your end, Marie):
@pwizla  Add a redirection here otherwise we'll get a 404 because the anchor changed